crates/autopilot/src/infra/shadow/orderbook/mod.rs

```diff
@@ -17,14 +17,15 @@ impl Orderbook {
     }
 
     /// Retrieves the current auction.
-    pub async fn auction(&self) -> reqwest::Result<domain::AuctionWithId> {
+    pub async fn auction(&self) -> anyhow::Result<domain::AuctionWithId> {
```

Comment:
The type of error is not important for the caller, so it is better to transform from `reqwest::Error` error to `anyhow::Error` than the other way around.